api/controllers/members.controller.ts

```diff
@@ -51,6 +51,24 @@ export class MembersController {
     }
   }
 
+  static async bulkCreate(req: Request, res: Response, next: NextFunction) {
+    try {
+      const mainService = new MainService();
+
+      await mainService.initalize(req.user._id);
+
+      const newMember = await mainService.runCommand(
+        "member",
+        "bulkCreate",
+        req.body
+      );
+
+      res.status(201).json(newMember);
```

Comment:
Bulk create should return the amount of correctly inserted rows, and ideally the number of wrong ones (if it can be determined, maybe bulk write in Mongo just writes or fails all)